Makefile

```diff
@@ -503,14 +503,14 @@ gen-doc =	\
 		else \
 			cd tools/doc && node ../../$(NPM) install; \
 		fi;\
-	[ -x $(NODE) ] && $(NODE) $(gen-json) || node
+	[ -x $(NODE) ] && $(NODE) $(1) || node $(1)
 
 out/doc/api/%.json: doc/api/%.md
-	$(gen-doc) $(gen-json)
+	$(call gen-doc, $(gen-json))
 
 # check if ./node is actually set, else use user pre-installed binary
```

Comment:
Just to clarify, in our `Makefile` we have two comments:

```
 # check if ./node is actually set, else use user pre-installed binary
```

Both of them are not in the correct place. Line 511 is definitely strange comment I don't see any check for `./node` there. Line 493 seems not correct as well since there are no checks too. But I see some checks in `get-doc`. So one comment should be removed and another one placed to the `gen-doc` definition.